File: src/main/java/br/com/jiratorio/domain/form/IssueForm.java
package br.com.jiratorio.domain.form;

import br.com.jiratorio.domain.dynamicfield.DynamicFieldsValues;
import lombok.Data;

import java.time.LocalDate;
import java.util.ArrayList;
import java.util.List;

@Data
public class IssueForm {

    private LocalDate startDate;

    private LocalDate endDate;

    private List<String> keys = new ArrayList<>();

    private List<String> systems = new ArrayList<>();

    private List<String> taskSize = new ArrayList<>();

    private List<String> epics = new ArrayList<>();

    private List<String> issueTypes = new ArrayList<>();

    private List<String> projects = new ArrayList<>();

    private List<String> priorities = new ArrayList<>();

    private List<DynamicFieldsValues> dynamicFieldsValues = new ArrayList<>();

}

File: src/main/java/br/com/jiratorio/domain/form/IssueForm.kt
package br.com.jiratorio.domain.form

import br.com.jiratorio.domain.dynamicfield.DynamicFieldsValues
import java.time.LocalDate
import java.util.ArrayList

data class IssueForm(

    var startDate: LocalDate? = null,

    var endDate: LocalDate? = null,

    var keys: MutableList<String?> = ArrayList(),

    var systems: MutableList<String?> = ArrayList(),

    var taskSize: MutableList<String?> = ArrayList(),

    var epics: MutableList<String?> = ArrayList(),

    var issueTypes: MutableList<String?> = ArrayList(),

    var projects: MutableList<String?> = ArrayList(),

    var priorities: MutableList<String?> = ArrayList(),

    var dynamicFieldsValues: MutableList<DynamicFieldsValues?> = ArrayList()

)
